extensions/amp-story/1.0/amp-story-grid-layer.js

```diff
@@ -134,6 +153,29 @@ export class AmpStoryGridLayer extends AmpStoryBaseLayer {
     });
   }
 
+  /**
+   * Styles text with a background color based on the value of
+   * the text-background-color attribute
+   * @private
+   */
+  setDescendentCssTextStyles_() {
+    const elementsToUpgradeStyles = scopedQuerySelectorAll(this.element,
+        TEXT_BACKGROUND_COLOR_SELECTOR);
+
+    Array.prototype.forEach.call(elementsToUpgradeStyles, element => {
+      const color = element.getAttribute(TEXT_BACKGROUND_COLOR_ATTRIBUTE_NAME);
+      const attributes = dict({
+        'style': `background-color:${color}`,
```

Comment:
@gmajoulet we already have a utility function that automatically picks a foreground text color based on a background color, right?  Do you think it would make sense to use that here?